files/index.html

```diff
@@ -0,0 +1,124 @@
+</head>
+<body class="theme-green">
+
+<div class="content first-content">
+
+
+  <div class="form">
+
+    <form id="domain-form">
+
+      <h2><label for="domain">
+        Enter a domain for the HSTS preload list:
+      </label></h2>
+
+      <input id="domain" type="text" placeholder="example.com" autofocus autocorrect="off" autocapitalize="off" spellcheck="false"></input>
+      <br>
+      <input id="check" type="submit" value="Check eligibility and status">
+    </form>
+
+    <div id="result-waiting" class="hidden">
+      <div id="spinner" class="spinner"></div>
+      <br>
+      <div id="checking"></div>
+    </div>
+
+    <div id="result" class="hidden">
+      <div id="summary"></div>
+      <div id="status"></div>
+      <div class="issues">
+        <div id="errors" class="issues-list"></div>
+        <div id="warnings" class="issues-list"></div>
+      </div>
+    </div>
+    
+    <form id="submit-form" class="hidden">
+    <hr>
+      <h2 id="submit-title">Submit</h2>
+      <div id="checkboxes">
+        <label>
+          <input type="checkbox" id="checkbox-owner"><span>I am the site owner of <tt><span class="domain-text">example.com</span></tt> or have their permission to preload HSTS.</span>
+        </label>
+        <span id="oops">
+        (If this is not the case, <tt><span class="domain-text">example.com</span></tt> may be sending a HSTS <tt>preload</tt> directive by accident. Please contact <a href="mailto:hstspreload@chromium.org">hstspreload@chromium.org</a> to let us know.)
+        </span>
+        <br><br>
+        <label>
+          <input type="checkbox" id="checkbox-subdomains"><span>I understand that preloading <span id="domain-3">example.com</span> will prevent all subdomains (<tt>*.<span class="domain-text">example.com</span></tt>) from being accessed without a valid HTTPS certificate.</span></label>
+      </div>
+      <br>
+      <input id="submit" type="submit" disabled
+         value="Submit to the HSTS Preload List">
+    </form>
+
+    <div id="submit-result" class="hidden">
+    </div>
+
+  </div>
+
+</div>
+
+<div class="content submission">
+
+  <div class="inner-content">
+
+    <h2>
+      Information
+    </h2>
+
+    <div id="text">
+      <p>This form is used to submit domains for inclusion in Chrome's <a href="https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security">HTTP Strict Transport Security (HSTS)</a> preload list. This is a list of sites that are hardcoded into Chrome as being HTTPS only. <a href="https://blog.mozilla.org/security/2012/11/01/preloading-hsts/">Firefox</a>, Safari, <a href="https://blogs.windows.com/msedgedev/2015/06/09/http-strict-transport-security-comes-to-internet-explorer-11-on-windows-8-1-and-windows-7/">IE 11 and Edge</a> also have HSTS preload lists which include the Chrome list. (See the <a href="http://caniuse.com/#feat=stricttransportsecurity">HSTS compatibility matrix</a>.)</p>
+
+    <h2>
+      Requirements
+    </h2>
```

Comment:
Same as above, single line? (In general you'll want to limit the vertical extent of the code)